k2/csrc/cuda/debug.cuh

```diff
@@ -0,0 +1,169 @@
+// k2/csrc/cuda/errors.h
+
```

Comment:
The convention is that we're just calling files .h even if they may contain cuda code, because due to the structure of the library almost every file can potentially instantiate cuda code even if it's not explicitly present.  I know this may seem odd, but for me it's convenient as my emacs recognizes as C++, and I think it will make the files seem less scary to people.

Reply:
Done.